tests/api/test_parameters.py

```diff
@@ -443,11 +446,8 @@ def test_nullable_parameter(simple_app):
     resp = app_client.put("/v1.0/nullable-parameters", content="null", headers=headers)
     assert resp.json() == "it was None"
 
-    resp = app_client.put("/v1.0/nullable-parameters", content="None", headers=headers)
```

Comment:
Why can we delete this one now?

Reply:
In query parameters it's impossible to represent `null` there:
- `?key=` could be interpreted as an empty string
- `?key` is not clear as discussed in this issue: 
https://github.com/spec-first/connexion/issues/1538

So connexion supports `"null"` and `"None"` as null values, so you can send:
- `?key=null`
- `?key=None`
(I'm actually not a fan of supporting `"None"` here, since it's Python specific, but if we still need to support `"null"`, the benefit of dropping it is probably too small.)

In json, null can be represented:
```
{
    "key": null
}
```
or for a `null` body (eg. try `json.dumps(None)`):
```
"null"
```
So we don't need to support `"None"` here. 
We actually also don't need to support `"null"` here, since we should always parse it before checking, in which case it's parsed to `None`. This is why I replaced all `is_null(body)` checks in the body validators with `body is None`.